i18n/i18n.go

```diff
@@ -84,7 +86,12 @@ func (i18n *I18n) T(locale, key string, args ...interface{}) string {
 		value = translations[translationKey].Value
 	} else {
 		// Save translations
-		i18n.SaveTransaltion(&Translation{Key: translationKey, Locale: locale, Backend: i18n.Backends[0]})
+		err := i18n.SaveTranslation(&Translation{Key: translationKey, Locale: locale, Backend: i18n.Backends[0]})
+		log.Printf("Error saving translation: [%s]: %s\n", locale, translationKey)
+		if err != nil {
+			delete(translations, translationKey)
```

Comment:
don't see the relation to the above comment. when i call `i18n.SaveTranslation()` and it failes it still somehow ends up having the key in the translations cache... that's why i added the `delete()... if err!=nil` - but i didnt see where it get's put in there.